accordion/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<title>
+    Accordion example
+</title>
+<head>
+    <style>
+        .accordion {
+            background-color: #f3e5f5;
+            color: #333;
+            cursor: pointer;
+            padding: 18px;
+            border: none;
+            text-align: left;
+            outline: none;
+            font-size: 15px;
+            transition: 0.4s;
+        }
+
+        /* .active, */
+        .accordion:hover {
+            background-color: #ce93d8;
+        }
+
+        /* guide to content entity: https://www.w3schools.com/cssref/css_entities.asp */
+        .accordion:after {
+            content: '\25BA';
+            color: #777;
+            font-weight: bold;
+            float: right;
+            margin-left: 5px;
+        }
+
+        .active:after {
+            content: "\25BC";
+        }
+
+        .section {
+            padding: 0 18px;
+            background-color: #f8f8f8;
+            display: none;
+            overflow: hidden;
+            transition: max-height 0.2s ease-out;
+        }
+    </style>
+</head>
+
+<body>
+    <h2>Accordion Example</h2>
+    <section>
+        <div class="accordion">Section 1</div>
+        <div class="section">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+                Curabitur eleifend
+                lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+                Pellentesque
+                facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+                posuere luctus
+                imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+                orci in,
+                condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+                lectus ac
+                vehicula
+            </p>
+        </div>
+    </section>
+    <section>
+        <div class="accordion">Section 2</div>
+        <div class="section">
```

Comment:
One thing to take care when you are making classes..avoid giving them the name of HTML tags.

Instead of '.section' use some more contextual class name.

Reply:
Thanks for letting me know. I changed the name and added a small margin to sections.  
![Screenshot_2019-10-22 Accordion example](https://user-images.githubusercontent.com/22999030/67254245-b7108e80-f499-11e9-8d9f-cb22a96690c9.png)